packages/quantic/cypress/e2e/default-1/results-per-page/results-per-page.cypress.ts

```diff
@@ -125,6 +125,16 @@ describe('quantic-result-per-page', () => {
       }
 
       describe('with invalid options', () => {
+        beforeEach(() => {
+          // This error occasionally occurs with the Salesforce Lightning Modal component, although we don't know exactly why it occurs, we do know that it only occurs in this Cypress test environment and never in the production environment.
```

Comment:
technically also occurs in the pager tests :P

Reply:
what do you mean?